File: 1894-merge-strings-alternately/merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        string result;
        int n=max(word1.size(),word2.size());
       
       
        for(int j=0;j<n;j++){
            if(j<word1.size()){
            result.push_back(word1[j]);}
            
             
            if(j<word2.size()){
            result.push_back(word2[j]);}
           
            
           
        }
        return result;
    }
};

File: 1894-merge-strings-alternately/merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        left=0
        right=0
        result=[]
        while left<len(word1) and right<len(word2):
            result.append(word1[left])
            result.append(word2[right])
            left+=1
            right+=1
        result.append(word1[left:])
        result.append(word2[right:])
                
                
        return "".join(result)
